src/Common/Core/Impl/Idle/IdleTimeAction.cs

```diff
@@ -4,10 +4,8 @@
 using System;
 using System.Collections.Generic;
 using Microsoft.Common.Core.Shell;
-using Microsoft.Languages.Core.Utility;
-using Microsoft.Languages.Editor.Shell;
 
-namespace Microsoft.Languages.Editor.Tasks {
+namespace Microsoft.Common.Core.Idle {
```

Comment:
Come to think of this, we may do another pass on core at some point and split it into real core and IDE stuff. For example, task list support is currently in the editor but task list is typically a global service. Anything can push items there: project, extensions, configuration management...